roles/multibench_run/README.md

```diff
@@ -0,0 +1,38 @@
+# Multi-bench role
+This role is a standard procedure to launch a [Multi-bench](https://github.com/perftool-incubator/crucible) test.
+It uses a pre-existing VM/host where all the crucible binaries are ready.
+
+## Requirements
+A PerformanceProfile must be correctly setup on your OCP before launching this playbook.
+The multibench_host should have been prepared before running this role [with this playbook](https://github.com/redhatci/ansible-collection-redhatci-ocp/playbooks/multibench_setup_host.yml).
+
+| Variable                  | Default                                                       | Type      | Required | Description                                                                                                  |
+|---------------------------|---------------------------------------------------------------|-----------|----------|--------------------------------------------------------------------------------------------------------------|
+| multibench_host           | null                                                          | string    | true     | Address of the multi-bench host accessible with SSH, should be define in the inventory                       |
```

Comment:
Is `multibench_host` var used in the playbook? I don't see any reference to it in the main playbook